src/token/handler.ts

```diff
@@ -406,6 +406,10 @@ export class RequestTokenHandler extends TokenHandler {
     this.connection.emit('databaseChange', token.newValue);
   }
 
+  onDatabaseMirroringPartner(token: DatabaseMirroringPartnerEnvChangeToken) {
+    this.connection.emit('databaseMirroringPartner', token.newValue);
+  }
```

Comment:
Similar to my previous question - the `ENVCHANGE` token for this is only supposed to be sent back by the server as a response to `Login7`. This adds a handler for general request response. I don't think this token should ever be received here. 🤔

Reply:
@arthurschreiber Ah, I had removed that in a commit that I hadn't yet pushed up. Just pushed the commit, the event is now only being handled within the `Login7` token handler. Good catch!